include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -407,8 +417,15 @@ padding:
     e6, e7, p5, p6, p7,
     ...]
 
-In addition to interval-padding pairs, this encoding requires an `order` to
-specify the logical tensor dimenions from the fastest-to slowest-varying.
+3.  Single interval-padding with linear component
+
+    #ttg.padded_shared<[2:+2] {offset = [[2], [1]]}>
+    [e0, e3, p0 p1,
+     e1, e3, p2, p3
+     ...]
+
```

Comment:
Can you explain how the `{offset = ..}` part are factored in to rearrange the elements? Also probably good to have another example to showcase 2-D.